src/snippets/switch.md

```diff
@@ -0,0 +1,57 @@
+---
+description: Switch Statement Snippet
+---
+
+# Switch
+
+Using `switch` statements in AssemblyScript
+
+```editor
+#!optimize=size&runtime=full
+export function switchSurprise(a: i32): i32 {
+  let response = -1;
+  switch(a) {
+    case 1:
+      response = 100;
+      break;
+    // Cases can also use braces
+    case 2: {
+      response = 200;
+      break;
+    }
+    case 3:
```

Comment:
This appears to mix braced / non-braced cases. Is this intended as part of the example? If not, I suggest to use braces everywhere so an eventual `let` or `const` variable declaration becomes correctly bound to the case, not the switch.

Reply:
This is intended but I'll leave a comment to specify that :smile: